packages/export-framework/start-server.js

```diff
@@ -19,6 +21,7 @@ module.exports = async ({
   coreConfig: incomingCoreConfig,
   port = env.PORT || 6008,
   exposedPort = env.EXPOSED_PORT || env.PORT || 6008,
+  exposedHost = env.EXPOSED_HOST || 'localhost',
```

Comment:
Since the env value is being set above there's no need to do the fallback to localhost now